trismart_smartboard_integration/models/smartboard_connector.py

```diff
@@ -23,6 +23,17 @@ def param_check(self, input_dict):
             error_dict = {"status": 400, "error": error, "odoo_customer_id": False}
         return [valid, error_dict]
 
+    def create_non_template_project (self, sb_lead_id):
+        project_env = self.env['project.project']
+        default_template_id = self.env.user.company_id.user_default_project_template.id or False
```

Comment:
We should user self.company_id.user_default_project_template.id for the current company instead self.env.user.company_id is the default company.